Blockzilla/DomainCompletion.swift

```diff
@@ -7,8 +7,9 @@ import AutocompleteTextField
 
 class DomainCompletion: AutocompleteTextFieldCompletionSource {
     private lazy var topDomains: [String] = {
-        let filePath = Bundle.main.path(forResource: "topdomains", ofType: "txt")
-        return try! String(contentsOfFile: filePath!).components(separatedBy: "\n")
+        let localeLang = Bundle.main.preferredLocalizations[0]
+        var fileName = localeLang != "en" ? "topdomains-" + localeLang : "topdomains"
```

Comment:
Ah, sorry – I suppose it largely is, exception to English.

Two improvements I'd like to see:
* Top domain files are in a folder to avoid polluting the existing folder
* English is not special cased (since it's "topdomains.txt" rather than "topdomains-en.txt" or whatever the locale code is)